### PR TITLE
setup add and remove

### PR DESCRIPTION
**Pull request by rmjspecial**
**Project Description**

- Add crud.js file.
- Add storage.js file.
- Add a clear.js file.
- Add function to add files to the to-do list.
- Add function to Edit files in to-do-list.

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      i.parentElement.classList.add('list-selected-item');
+    });
+  });
+
+  const remove = document.querySelectorAll('.fa-trash-alt');
+  remove.forEach((i) => {
+    i.addEventListener('click', () => {
+      deleteTask(i.parentElement);
+    });
+  });
```
Comment (AmaduKamara) [inline src/crud.js:60]: When I tick a todo as completed and click on the delete icon, the todo is removed from the screen, and after that, when I refresh the browser the same todo that's supposed to have been deleted still shows up on the list again.

- [x] Kindly work on the delete task so we can be able to delete a todo, it gets removed from the list on the frontend and as well from localStorage. So when we refresh the browser, the remaining todos are fetched and get populated on the screen or zero todos in a case where there was only one. 👍
- [x] Make sure when we delete a todo, you update the localStorage list. 

Before delete

image

After delete, the same todo shows again

image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @rmjspecial,

AlonzoLeroneThisCloseGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly
✅ No linter error
✅ Good PR description
✅ Adding, displaying, and updating a todo work fine
✅ Correctly indexing todos 👍

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly rework the delete functionality and correctly update your localStorage after deletion 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by rmjspecial** [comment]
fix delete issue.

**Review by Gambit142 — APPROVED** [review]
Hi @rmjspecial 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Great work making changes requested by the previous reviewer

ApprovedAgreeGIF


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**